### PR TITLE
GetData arguments tests

### PR DESCRIPTION
**Pull request by hcorg**

### CODE REVIEW

**Comment by hcorg** [comment]
I'm not validating "to" timestamp. As it is excluded from range, I assume it can be anything. (`GetData(t0, t0 + 1ms)` could be an idiom for 'get single sample')

File: SignalsIntegrationTests/SignalsIntegrationTests/GetDataArgumentsTests.cs [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SignalsIntegrationTests.Infrastructure;
+
+namespace SignalsIntegrationTests
+{
+    [TestClass]
+    public class GetDataArgumentsTests : TestsBase
+    {
+        private DateTime FromTimestamp { get; } = new DateTime(2029, 1, 1, 0, 0, 0, 0);
+        private DateTime ToTimestamp { get; } = new DateTime(2029, 1, 8, 0, 0, 0, 0);
```
Comment (bartoszkp) [inline SignalsIntegrationTests/SignalsIntegrationTests/GetDataArgumentsTests.cs:13]: Perhaps we should add one test case with `FromTimestamp` > `ToTimestamp` (I'm guessing that the repository would yield empty result, and that seems an acceptable convention)?

Comment (hcorg) [inline SignalsIntegrationTests/SignalsIntegrationTests/GetDataArgumentsTests.cs:13]: I think we should add separate issue for that, and test it per `MissingValuePolicy`. All those "download one sample older than from" etc. might result in some strange behavior in possible implementations.

Our is safe, thanks to TImeEnumerator :D

Comment (bartoszkp) [inline SignalsIntegrationTests/SignalsIntegrationTests/GetDataArgumentsTests.cs:13]: You're right.
